### PR TITLE
Add ui for manager and server information

### PR DESCRIPTION
**Pull request by maffoo**
This is built on the changes in #5, so I'll rebase and merge after that is in master. For now, just look at the last two commits. Also, there's still a lot to be done to style the UI and make everything pretty; for now, I just want to get the structure in place to have feature-parity with the old GWT client.

### CODE REVIEW

**Comment by joshm-3b6b58** [comment]
I'll merge my last pull request right now to simplify this, then I'll take a look

File: client-js/app/elements/labrad-manager.html [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        <th>Type</th>
+        <th>Name</th>
+        <th>Server Reqs</th>
+        <th>Server Reps</th>
+        <th>Client Reqs</th>
+        <th>Client Reps</th>
+        <th>Messages Sent</th>
+        <th>Messages Recv</th>
+      </tr>
+      <template is="dom-repeat" items="{{connections}}">
```
Comment (joshm-3b6b58) [inline client-js/app/elements/labrad-manager.html]: I tried makign this comment, but it didn't seem to take, so my apologies if this comment shows up twice:

I would use 

```
<thead>
  <th>Id</th>
 ...
</thead>
<tbody>
  <tr>
    <td>
    ...
</body
```

This gives separate CSS attributes we can mess with later when rendering the table, you can take a look at how I was rendering the registry table in the branch `u/joshmutus/html`

**Comment by joshm-3b6b58** [comment]
Ran the branch on my machine. LGTM other than the aforementioned table tagging.

File: client-js/app/elements/labrad-manager.html (hunk 1 above)
Comment (maffoo) [inline client-js/app/elements/labrad-manager.html]: done.

**Pull request merged**